cmd/catalog.go

```diff
@@ -233,15 +233,37 @@ This describes the service and the associated plans.
 
 		if len(plans) > 0 {
 			table.Append([]string{"PLANS"})
+
 			for _, plan := range plans {
+
+				// create the display values for required  and optional parameters
+				requiredParameterNames := []string{}
+				optionalParameterDisplay := []string{}
+				for _, parameter := range plan.Parameters {
+					if parameter.Required {
+						requiredParameterNames = append(requiredParameterNames, parameter.Name)
+					} else {
+						// until we have a better solution for displaying the plan data (like a separate table perhaps)
+						// this is simplest thing to do
+						defaultValueStr := fmt.Sprintf("%v", parameter.DefaultValue)
+						if len(defaultValueStr) > 0 {
```

Comment:
This causes an issue: `mysql-persistent` has an empty string as default value for several properties (e.g. `MYSQL_PASSWORD`) which appear here as not having a default value. So I think `ServicePlanParameter` should record that information so that more accurate information is displayed.

Reply:
Fixed